docker/docker-compose.arm64.yml

```diff
@@ -1,11 +0,0 @@
-version: '3.7'
-
-services:
-  mssql:
-    image: mcr.microsoft.com/azure-sql-edge:latest
```

Comment:
Is overriding the SQL Server image not necessary on ARM anymore?

Reply:
It's been a while indeed, like said in description.

More info in https://devblogs.microsoft.com/azure-sql/development-with-sql-in-containers-on-macos/